CELLULAR_NETWORK_FAILED_REASON and CELLULAR_NETWORK_RADIO_TYPE not defined

Hello Everyone,

I am using "gomavlib" in my project. But, I've met an issue recently.
Does anyone know how to solve the problem described below?

PS agent\cmd> go run .\main.go
# github.com/aler9/gomavlib/pkg/dialects/common
..\..\..\..\..\..\go\pkg\mod\github.com\aler9\gomavlib@v0.0.0-20211014051456-5b4257e7e03a\pkg\dialects\common\dialect.go:17730:16: undefined: CELLULAR_NETWORK_FAILED_REASON
..\..\..\..\..\..\go\pkg\mod\github.com\aler9\gomavlib@v0.0.0-20211014051456-5b4257e7e03a\pkg\dialects\common\dialect.go:17732:7: undefined: CELLULAR_NETWORK_RADIO_TYPE

--- a/message_definitions/v1.0/common.xml
+++ b/message_definitions/v1.0/common.xml
@@ -3670,6 +3670,29 @@
         <description>One or more packet data bearers is active and connected</description>
       </entry>
     </enum>
+    <enum name="CELLULAR_NETWORK_FAILED_REASON">
+      <description>These flags are used to diagnose the failure state of CELLULAR_STATUS</description>
+      <entry value="0" name="CELLULAR_NETWORK_FAILED_REASON_NONE">
+        <description>No error</description>
+      </entry>
+      <entry value="1" name="CELLULAR_NETWORK_FAILED_REASON_UNKNOWN">
+        <description>Error state is unknown</description>
+      </entry>
+      <entry value="2" name="CELLULAR_NETWORK_FAILED_REASON_SIM_MISSING">
+        <description>SIM is required for the modem but missing</description>
+      </entry>
+      <entry value="3" name="CELLULAR_NETWORK_FAILED_REASON_SIM_ERROR">
+        <description>SIM is available, but not usuable for connection</description>
+      </entry>
+    </enum>
+    <enum name="CELLULAR_NETWORK_RADIO_TYPE">
+      <description>Cellular network radio type</description>
+      <entry value="0" name="CELLULAR_NETWORK_RADIO_TYPE_NONE"/>
+      <entry value="1" name="CELLULAR_NETWORK_RADIO_TYPE_GSM"/>
+      <entry value="2" name="CELLULAR_NETWORK_RADIO_TYPE_CDMA"/>
+      <entry value="3" name="CELLULAR_NETWORK_RADIO_TYPE_WCDMA"/>
+      <entry value="4" name="CELLULAR_NETWORK_RADIO_TYPE_LTE"/>
+    </enum>
     <enum name="PRECISION_LAND_MODE">
       <description>Precision land modes (used in MAV_CMD_NAV_LAND).</description>
       <entry value="0" name="PRECISION_LAND_MODE_DISABLED">

--- a/message_definitions/v1.0/development.xml
+++ b/message_definitions/v1.0/development.xml
@@ -89,29 +89,6 @@
         <param index="1" label="Action" enum="FENCE_ACTION">Fence action on breach.</param>
       </entry>
     </enum>
-    <enum name="CELLULAR_NETWORK_RADIO_TYPE">
-      <description>Cellular network radio type</description>
-      <entry value="0" name="CELLULAR_NETWORK_RADIO_TYPE_NONE"/>
-      <entry value="1" name="CELLULAR_NETWORK_RADIO_TYPE_GSM"/>
-      <entry value="2" name="CELLULAR_NETWORK_RADIO_TYPE_CDMA"/>
-      <entry value="3" name="CELLULAR_NETWORK_RADIO_TYPE_WCDMA"/>
-      <entry value="4" name="CELLULAR_NETWORK_RADIO_TYPE_LTE"/>
-    </enum>
-    <enum name="CELLULAR_NETWORK_FAILED_REASON">
-      <description>These flags are used to diagnose the failure state of CELLULAR_STATUS</description>
-      <entry value="0" name="CELLULAR_NETWORK_FAILED_REASON_NONE">
-        <description>No error</description>
-      </entry>
-      <entry value="1" name="CELLULAR_NETWORK_FAILED_REASON_UNKNOWN">
-        <description>Error state is unknown</description>
-      </entry>
-      <entry value="2" name="CELLULAR_NETWORK_FAILED_REASON_SIM_MISSING">
-        <description>SIM is required for the modem but missing</description>
-      </entry>
-      <entry value="3" name="CELLULAR_NETWORK_FAILED_REASON_SIM_ERROR">
-        <description>SIM is available, but not usuable for connection</description>
-      </entry>
-    </enum>
     <enum name="MAV_CMD">
       <entry value="247" name="MAV_CMD_DO_UPGRADE" hasLocation="false" isDestination="false">
         <description>Request a target system to start an upgrade of one (or all) of its components.

